R/print_anova.R

```diff
@@ -7,6 +7,7 @@
 #' @param x Data.frame. A \code{data.frame} of class \code{apa_variance_table} as returned by \code{\link{arrange_anova}}.
 #' @param intercept Logical. Indicates if intercept test should be included in output.
 #' @param es Character. The effect-size measure to be calculated; can be either \code{ges} for generalized eta-squared, \code{pes} for partial eta-squared or \code{es} for eta-squared.
+#'   Be careful when calculating eta-squared: It is only calculated correctly if the design is balanced.
```

Comment:
Could we reiterate the contents of the message given by the function here?

Reply:
I like that idea.